src/BuildScriptGenerator/DotNetCore/DotnetCorePlatform.cs

```diff
@@ -139,12 +139,20 @@ public PlatformDetectorResult Detect(RepositoryContext context)
                 manifestFileProperties[ManifestFilePropertyKeys.DotNetCoreSdkVersion] = globalJsonSdkVersion;
             }
 
+            var oryxAppTypeEnvironmentVar = Environment.GetEnvironmentVariable("Oryx_App_Type");
```

Comment:
We do not want to read environment variables directly this way because users should be able to set this via `build.env` file or environment variables...look how we do for `PROJECT` setting key for example...basically this should be accessible via options api

Reply:
Also let us validate the supplied value for app type and maybe write a warning saying it is not supported and list the supported app types.